string.rst

```diff
@@ -607,6 +607,23 @@ Convert Slack short codes to emojis with the ``slack-emoji`` locale::
     $transliterator->transliterate('Menus with :green_salad: or :falafel:');
     // => 'Menus with 🥗 or 🧆'
 
+Inverse Emoji Transliteration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 7.1
+
+    The inverse emoji transliteration was introduced in Symfony 7.1.
+
+Given the textual representation of an emoji, you can reverse it back to get the
+actual emoji thanks to the :ref:`emojify filter <reference-twig-filter-emojify>`:
+
+.. code-block:: twig
+
+    {{ 'green heart'|emojify }}            {# renders: 💚 #}
```

Comment:
I tested it, and it does not work. it should be
```suggestion
    {{ ':green-heart:'|emojify }}            {# renders: 💚 #}
```

Reply:
Thanks for the review! Fixed.